mobile/src/main/java/org/hawkular/client/android/service/AcknowledgeAlertService.java

```diff
@@ -0,0 +1,21 @@
+package org.hawkular.client.android.service;
+
+import org.hawkular.client.android.backend.model.Trigger;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.FormUrlEncoded;
+import retrofit2.http.POST;
+
+/**
+ * Created by pallavi on 26/06/17.
+ */
+
+public interface AcknowledgeAlertService {
```

Comment:
Don't make new service for every call. we will have few services like `Trigger` `Alert` `Metric` and so on. Rest of it will be on methods inside them.